cassdegrees/templates/list.html

```diff
@@ -58,29 +60,28 @@
                                     {% if key != "id" %}
                                         <td>{{ item }}</td>
                                     {% else %}
-                                        <td><input title="Select" type="radio" name="id" value="{{ item }}"  /></td>
-                                        <td>(<a href="/api/model/{{ title.lower }}/{{ item }}/">View</a>)</td>
+                                        <td><input title="Select" type="checkbox" name="id" value="{{ item }}"  /></td>
+                                        <td>
+                                            (<a href="/api/model/{{ title.lower }}/{{ item }}/">View</a>)
```

Comment:
Would be good if this appeared as the button below, but I don't know how to do this while making it inline

Reply:
I'll have a look into it tomorrow.
But do you mean one column where they're on top of each other?
So:
| View |
| Edit  |